shared/metrics/src/main/java/io/pravega/shared/metrics/OpStatsLoggerImpl.java

```diff
@@ -27,18 +29,18 @@
     private final String successName;
     private final Timer fail;
     private final String failName;
-    private final MetricRegistry metricRegistry;
+    private final MeterRegistry metricRegistry;
 
     //endregion
 
     //region Constructor
 
-    OpStatsLoggerImpl(MetricRegistry metricRegistry, String baseName, String statName) {
+    OpStatsLoggerImpl(MeterRegistry metricRegistry, String baseName, String statName, String... tags) {
         this.metricRegistry = Preconditions.checkNotNull(metricRegistry, "metrics");
-        this.successName = name(baseName, statName);
-        this.failName = name(baseName, failMetricName(statName));
-        this.success = this.metricRegistry.timer(this.successName);
-        this.fail = this.metricRegistry.timer(this.failName);
+        this.successName = baseName + "." + statName;
+        this.failName = baseName + "." + failMetricName(statName); //TODO: replace the original name() method
```

Comment:
Make sure to incorporate the guidelines in your code changes:

https://github.com/pravega/pravega/wiki/Contributing#todos

Reply:
Yes, I'm actively working on the TODOs.
I believe that you encouraged me to have the minimum essential changes out asap, so the team could have sufficient time to review it.
This TODOs is also my approach to highlight the critical places for discussion.
I'll ensure the final commit will stick with the guidelines strictly.